docs/sphinx/manual/hpx-functions.rst

```diff
@@ -0,0 +1,82 @@
+..
```

Comment:
Could you move this content to https://github.com/STEllAR-GROUP/hpx/blob/master/docs/sphinx/manual/writing_single_node_hpx_applications.rst?